part-time-schedule/week-1.md

```diff
@@ -0,0 +1,87 @@
+| ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| BIP0032                                                            | [https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)                                                                       |
+| TABConf 2021 Building Your own bitcoin wallet with BitcoinJS       | [https://youtu.be/Bwz2P2hPVpk](https://youtu.be/Bwz2P2hPVpk)                                                                                                                                           |
+| Bitstein - Setting up a bitcoin lightning network test environment | [https://medium.com/@bitstein/setting-up-a-bitcoin-lightning-network-test-environment-ab967167594a](https://medium.com/@bitstein/setting-up-a-bitcoin-lightning-network-test-environment-ab967167594a) |
+
+### Exercises
+
+1. Read the Mastering Lightning book appendix on bitcoin transactions.
+   * The username:password to access the book is: `qala:lightning`
+1. Read about Hierarchical Deterministic Wallets in BIP0032.
+   * You do not need to read/understand the child key derivation functions.
+   * You should conclude with a strong understanding of how parent and child keys are derived from each other, and from a master seed .
+1. Run through the TABconf wallet demo video found in the link above, following along with the demonstration.
+1. Set up a manual bitcoin and lightning developer environment on regtest by following the guide from Bitstein: [Setting up a Bitcoin/Lightning test environment](https://medium.com/@bitstein/setting-up-a-bitcoin-lightning-network-test-environment-ab967167594a)
+   * Note, this environment should ideally be built using Bitcoin Core and LND which have been build from source by yourself.
+1. Set up a simulated Lightning network with Polar using at least three lightning nodes with two channels connecting them.
+   * Try create some invoices and paying them. See how the balances of the channels change.
+   * Play around a bit and expand your network. Try adding some more nodes.
+* Run all Bitcoin Core unit tests
+* Choose area of the codebase you're interested in, pick a functional test that covers it, and then run that test
+  * hint: see documentation in `test/README.md` for clues on how to run individual tests)
+
+### Deliverables
+
+1. TABConf: please send us file: `src/util/bitcoinjs-lib.ts`.
+1. Screenshot of your lightning developer environment resulting from the guide from Bitstein.
+1. Screenshot of your simulated lightning network on Polar.
+1. A P2WPKH transaction in hex format
+1. A brief explanation on what the miner simulation is able to simulate
```

Comment:
I'd recommend taking out all the miner stuff, just to keep this week more focused.

Reply:
Agreed

Reply:
Oh yeah I took out the rest and definitely missed this. Thanks!